app/controllers/admin/content_controller.rb

```diff
@@ -113,6 +113,19 @@ def autosave
     render :text => nil
   end
 
+  def merge
+# find the ID of the article the admin submitted
+    @article1 = Article.find(params[:merge_with])
```

Comment:
I was thinking I would use article1 in one of my views, but not article2, because I would be merging that one in. But I never got around to it.